yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -374,7 +374,39 @@ Internet-Draft                 YANG Semver                  October 2022
    modifier strings.  E.g., artifact version "1.2.3_non_compatible" MUST
    NOT be defined if artifact version "1.2.3" has already been defined.
 
-3.3.1.  YANG Semver with submodules
+3.3.1.  Branching Limitations with YANG Semver
+
+   YANG artifacts that use the YANG Semver revision-label scheme MUST
+   ensure that two artifacts with the same MAJOR version number and no
+   _compatible or _non_compatible modifiers are backwards compatible.
+   Therefore, certain branching schemes cannot be used with YANG Semver.
+   For example, the following branching scheme cannot use YANG Semver
+   for its revision-label scheme:
+
+
+
+
+
+
+
+Clarke, et al.            Expires 26 April 2023                 [Page 7]
+
+Internet-Draft                 YANG Semver                  October 2022
+
+
+         3.5.0 -- 3.6.0 (add leaf foo)
+           |
+           |
+         3.20.0 (added leaf bar)
+
+   In this case, one would assume 3.20.0 is backwards compatible with
+   3.6.0 (per YANG Semver rules), but that would not be the case if leaf
+   foo is assumed to work in 3.20.0.
+
+   Note that this type of branching is possible.  However, it cannot use
+   YANG Semver as its revision-label scheme.
+
```

Comment:
Note that this type of branched parent-child relationship, where two revisions have different backwards compatible changes based on the same parent, is allowed in [ref Module Versioning].